app/main.js

```diff
@@ -1,163 +1,229 @@
+    label: 'Minimize',
+    accelerator: 'CmdOrCtrl+M',
+    role: 'minimize'
+  }]
+}, {
+  label: 'Help',
+  role: 'help',
+  submenu: [{
+    label: "Summoner's War Exporter GitHub",
+    click: function () {
+      shell.openExternal('https://github.com/Xzandro/sw-exporter')
+    }
+  }]
+}]
+
+function findReopenMenuItem () {
+  const menu = Menu.getApplicationMenu()
+  if (!menu) return
+
+  let reopenMenuItem
+  menu.items.forEach(function (item) {
+    if (item.submenu) {
+      item.submenu.items.forEach(function (item) {
+        if (item.key === 'reopenMenuItem') {
+          reopenMenuItem = item
+        }
+      })
+    }
+  })
+  return reopenMenuItem
+}
+
+if (process.platform === 'darwin') {
+  const name = app.getName()
+  template.unshift({
+    label: name,
+    submenu: [{
+      label: `About ${name}`,
+      role: 'about'
+    }, {
+      type: 'separator'
+    }, {
+      label: `Hide ${name}`,
+      accelerator: 'Command+H',
+      role: 'hide'
+    }, {
+      label: 'Hide Others',
+      accelerator: 'Command+Alt+H',
+      role: 'hideothers'
+    }, {
+      label: 'Show All',
+      role: 'unhide'
+    }, {
+      type: 'separator'
+    }, {
+      label: 'Quit',
+      accelerator: 'Command+Q',
+      click: function () {
+        app.quit();
+      }
+    }]
+  });
+
+}
+
 let defaultConfig = {
-  Config: {
-    App: { filesPath: defaultFilePath, debug: false, clearLogOnLogin: false },
-    Proxy: { port: 8080, autoStart: false },
-    Plugins: {}
-  }
+    Config: {
+        App: {
+            filesPath: defaultFilePath,
+            debug: false,
+            clearLogOnLogin: false
+        },
+        Proxy: {
+            port: 8080,
+            autoStart: false
+        },
+        Plugins: {}
+    }
 }
 let defaultConfigDetails = {
-  ConfigDetails: {
-    App: { debug: { label: 'Show Debug Messages' }, clearLogOnLogin: { label: 'Clear Log on every login'} },
-    Proxy: { autoStart: { label: 'Start proxy automatically' } },
-    Plugins: {}
-  }
+    ConfigDetails: {
+        App: {
+            debug: {
+                label: 'Show Debug Messages'
+            },
+            clearLogOnLogin: {
+                label: 'Clear Log on every login'
+            }
+        },
+        Proxy: {
+            autoStart: {
+                label: 'Start proxy automatically'
+            }
+        },
+        Plugins: {}
+    }
 }
-
 global.plugins = [];
-
 app.on('ready', () => {
-  createWindow();
-
-  storage.getAll(function (error, data) {
-    if (error) throw error;
-
-    global.config = _.merge(defaultConfig, data);
-    global.config.ConfigDetails = defaultConfigDetails.ConfigDetails;
-
-    fs.ensureDirSync(global.config.Config.App.filesPath);
-
-    global.plugins = loadPlugins();
-
-    if (global.config.Config.Proxy.autoStart) {
-      proxy.start(config.Config.Proxy.port);
-    }
-  });
+  const menu = Menu.buildFromTemplate(template)
+  Menu.setApplicationMenu(menu)
+    createWindow();
```

Comment:
Tried to get style to match as closely as possible, I think I may have accidentally hit the key command to beautify on accident.